### ISSUE TITLE
Develop content to inform users of any scheduled maintenance windows or anticipated downtime of back end system - must happen before launch

### ISSUE THREAD

**Issue by leannammiller** [department-of-veterans-affairs/vets-website#3578]
pre-launch for education benefits: https://github.com/department-of-veterans-affairs/edu-benefits-team/blob/master/project-management-and-charter-doc/checklist.md

**Comment by akainic** [comment]
Duplicate of https://github.com/department-of-veterans-affairs/edu-benefits-team/issues/101. Closing this one.
